### DISCUSSION TITLE
BUG: 2e Intelligence Track selects Science Track

### DISCUSSION THREAD

**Discussion by lilChopper83** [bcholmes/StarTrek2d20#244] [General]
In the Career section of the character generator, when you select "Intelligence" it brings up "Science Track" instead. I only have 2E Core selected.

**Comment by bcholmes** [comment]
Thanks for reporting this. Someone on the Modiphius forums said something similar, but I confess that I haven't been able to reproduce this behaviour, myself. For me, the Intelligence option always brings up Intelligence.

**Reply by lilChopper83** [reply]
Interesting, I was able to reproduce it initially, but I'll try again and see if I can narrow down the steps I took.

**Reply by lilChopper83** [reply]
![Screenshot 2024-12-27 13 41 42](https://github.com/user-attachments/assets/a6f85a75-964e-451d-a33d-0f96e72f20ea)
Yep, can confirm, I tried various different combinations of decisions leading up to "career" and every time it displays Science Track selections instead of Intelligence. To clarify the title is Intelligence and it is just the Primary and Secondary department selections and the spiel at the top that are incorrect.

**Reply by bcholmes** [reply]
Ah. Thanks for that. I can see the problem now. Should be fixed in the latest version.
